primo-explore/custom/01NWU_INST-HSLV/css/custom1.css

```diff
@@ -1800,3 +1802,8 @@ prm-atoz-search-bar {
 #libchat_button a:hover{
   box-shadow: none;
 }
+
+/* Set the facets font to normal */
+.section-content .md-chips .md-chip.wrapping-chip strong.facet-title.layout-row {
+  font-weight: normal !important;
```

Comment:
Seems odd that ever other style is able to override, but not this one. It might be worth dropping a comment explaining the why behind this, but on the flip side I think most people who use CSS will be aware that this is the big guns solution. 🤷

Reply:
It turns out Bart also copied and pasted someone else's suggested solution: https://primoviews.org/css-snippets/#facets 
Removing `!important` also removed the intended styling. I changed the CSS classes to apply the styling on the target element.